source/app/documentmanager.cpp

```diff
@@ -24,17 +24,22 @@ DocumentManager::DocumentManager()
 {
 }
 
-Document &DocumentManager::addDocument()
+Document &DocumentManager::addDocument(
+        const SettingsManager &settings_manager)
 {
     myDocumentList.emplace_back(new Document());
     myCurrentIndex = static_cast<int>(myDocumentList.size()) - 1;
+
+    auto settings = settings_manager.getReadHandle();
+    myDocumentList.back()->getViewOptions().setZoom(settings->get(Settings::LastZoomLevel));
```

Comment:
I'm not so sure about applying the setting here - for a "default" document we should be ok. On the other hand, at some point in the future we might find it useful to serialise the zoom level with a file...

Reply:
At this point, at least, I'm thinking it probably won't be likely that we want to save the zoom level with a file, since that's specific to a user's machine / monitor?
More likely might be saving per-document zoom settings locally, but I think the current behaviour is good for now.

Reply:
Yep, I misused the idea of serialising. I was thinking precisely for a per-document zoom setting locally